src/components/atoms/AdminButton/AdminButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { ButtonHTMLAttributes, RefObject } from "react";
```

Comment:
This component, since converted to tailwind, should probably be moved to `./src/components/admin/Button` so we'll know we already have it refactored.

And I do think some other PR had done it in similar manner, so whichever PR gets to staging second, can merge from both.

Reply:
moved to `src/admin/Button`
https://github.com/sparkletown/sparkle/pull/2812/commits/0f78efe3a381196c9e5b7cacf68aa5c14daa88bc